integration/scripts/README.mkd

```diff
@@ -0,0 +1,83 @@
+Scripts
+=======
+
```

Comment:
Since this in an open source project, I think it'd be worth mentioning here that these are to be used while on Puppet's internal network. If we specifically don't do this on purpose or as policy, that's fine.